### ISSUE TITLE
model.compile(loss={'ctc': lambda y_true, y_pred: y_pred}, optimizer='adam', metrics=['accuracy'])

### ISSUE THREAD

**Issue by zfxxfeng** [YCG09/chinese_ocr#349]
model.compile(loss={'ctc': lambda y_true, y_pred: y_pred}, optimizer='adam', metrics=['accuracy'])
你好，想问一下，train.py中，168行，这里的loss，返回的为什么是 lambda y_true, y_pred: y_pred。这个的意思不就是返回y_pred的值呢。
这里不是应该写model.compile(loss={'ctc':loss_out}, optimizer='adam', metrics=['accuracy'])
期待您的回答

**Comment by LiuXiao617111** [comment]
@zfxxfeng  l have same question. If you have answer,  please share it with me.  
thanks.

**Comment by LiuXiao617111** [comment]
@zfxxfeng  同问。如果知道原因麻烦分享一下。
